### ISSUE TITLE
Primary image style

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tripodtr/npr-android-app#149]
```
Would like to increase the size of the primary photo. Need designs first.

```

Original issue reported on code.google.com by `jpenn...@gmail.com` on 12 Apr 2012 at 8:45

**Comment by GoogleCodeExporter** [comment]
```
Designs should be included in forthcoming tablet-centric work currently in 
progress.
```

Original comment by `jpenn...@gmail.com` on 18 Sep 2012 at 7:45
- Changed state: **Started**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.4 update.
```

Original comment by `jpenn...@gmail.com` on 17 Oct 2012 at 10:38
- Changed state: **Live**
